### PR TITLE
[Feature][e2e] project management test cases

### PR DESCRIPTION
**Pull request by yangyunxi**
relative issue: #7328 

add project management test cases
create project, project name and description

### CODE REVIEW

**Comment by davidzollo** [comment]
good job, @kezhenxu94 please take a look, thx

File: dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/pages/project/ProjectPage.java [hunk 1]
```diff
@@ -65,36 +69,50 @@ public ProjectPage create(String project) {
         createProjectForm().buttonSubmit().click();
 
         new WebDriverWait(driver(), 10)
-            .until(ExpectedConditions.textToBePresentInElementLocated(By.className("project-name"), project));
+                .until(ExpectedConditions.textToBePresentInElementLocated(By.className("project-name"), project));
+
+        return this;
+    }
+
+    public ProjectPage create(String project, String describe) {
```
Comment (kezhenxu94) [inline dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/pages/project/ProjectPage.java:77]: Project description is not that important and we don't want to cover every aspect of the ui so I think we don't need the test of creating project with description. It costs resources without bringing too many benefits.

**Review by kezhenxu94 — CHANGES_REQUESTED** [review]

File: dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/pages/project/ProjectPage.java (hunk 1 above)
Comment (yangyunxi) [reply dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/pages/project/ProjectPage.java:77]: > Project description is not that important and we don't want to cover every aspect of the ui so I think we don't need the test of creating project with description. It costs resources without bringing too many benefits.

E2E task has this item, so is there any E2E task that can be done, please give it?

**Comment by kezhenxu94** [comment]
No update for a long time, please reopen if you want to continue

**Pull request closed without merging**